src/Musl/utils.jl

```diff
@@ -71,11 +63,10 @@ will give an incorrect result for the case `x = y = 0x1.800000e000001p+0`.
 # should probably check with LLVM, see #9855.
 """
     is_fma_fast(T)
-
```

Comment:
I see, fixing merge conflicts was a pain, I'm not sure why this happened, since I never touched this file.